client/my-sites/plan-features-2023-grid/index.tsx

```diff
@@ -495,34 +496,38 @@ export class PlanFeatures2023Grid extends Component<
 	}
 
 	renderPlanPrice( planPropertiesObj: PlanProperties[], options?: PlanRowOptions ) {
-		const { isReskinned, isLargeCurrency } = this.props;
+		const { isReskinned, isLargeCurrency, is2023OnboardingPricingGrid, translate } = this.props;
 
-		return planPropertiesObj
-			.filter( ( { isVisible } ) => isVisible )
-			.map( ( properties ) => {
-				const { planName, rawPrice } = properties;
-				const classes = classNames( 'plan-features-2023-grid__table-item', 'is-bottom-aligned', {
-					'has-border-top': ! isReskinned,
-				} );
-				const hasNoPrice = rawPrice === undefined || rawPrice === null;
-
-				return (
-					<Container
-						scope="col"
-						key={ planName }
-						className={ classes }
-						isMobile={ options?.isMobile }
-					>
-						{ ! hasNoPrice && (
-							<PlanFeatures2023GridHeaderPrice
-								planProperties={ properties }
-								is2023OnboardingPricingGrid={ true }
-								isLargeCurrency={ isLargeCurrency }
-							/>
-						) }
-					</Container>
-				);
+		return planPropertiesObj.map( ( properties ) => {
+			const { planName, rawPrice } = properties;
+			const isWooExpressPlus = isWooExpressPlusPlan( planName );
+			const classes = classNames( 'plan-features-2023-grid__table-item', 'is-bottom-aligned', {
+				'has-border-top': ! isReskinned,
 			} );
+			const hasNoPrice = rawPrice === undefined || rawPrice === null;
+
+			return (
+				<Container
+					scope="col"
+					key={ planName }
+					className={ classes }
+					isMobile={ options?.isMobile }
+				>
+					{ ! hasNoPrice && (
+						<PlanFeatures2023GridHeaderPrice
+							planProperties={ properties }
+							is2023OnboardingPricingGrid={ is2023OnboardingPricingGrid }
```

Comment:
Thanks for fixing it! The value passed to the `PlanFeatures2023GridHeaderPrice` component should be `true`. The `is2023OnboardingPricingGrid` prop is never passed to this component despite being present in the type definition.

I have a fix for that issue here: https://github.com/Automattic/wp-calypso/pull/75826

Reply:
Ah, thanks for catching that as well! I'll get that taken care of shortly.